docs/source/advanced/mixed_precision.rst

```diff
@@ -5,49 +5,130 @@
 
 .. _amp:
 
-Mixed Precision Training
-========================
-
-Mixed precision combines the use of both FP32 and lower bit floating points (such as FP16) to reduce memory footprint during model training, resulting in improved performance.
 
-Lightning offers mixed precision training for GPUs and CPUs, as well as bfloat16 mixed precision training for TPUs.
+#########
+Precision
+#########
+
+There are numerous benefits to using numerical formats with lower precision than the 32-bit floating-point or higher precision such as 64-bit floating-point.
+
+Lower precision, such as the 16-bit floating-point, requires less memory, enabling the training and deployment of larger neural networks. Second, they require less memory bandwidth, thereby speeding up data transfer operations. Third, math operations run much faster in reduced precision, especially on GPUs with Tensor Core support for that precision.
```

Comment:
This requires citation or needs to be rewritten. 
Let me know if you need help with it.

Reply:
added! mind check if it's the correct way?

Reply:
If the text is taken word by word, it has to be put in quotation marks. The reference alone is not enough. My suggestion is to formulate this in our own words and add the reference. One way to do this is to summarize in fewer words. The second best option is to add the quotation marks.

Reply:
rephrased!